src/styles/main.scss

```diff
@@ -1,3 +1,5 @@
+@import "utils/varaibles";
```

Comment:
There is a typo in the imported file name. It should be 'variables' instead of 'varaibles'.